yt/yt/library/containers/cri/config.cpp

```diff
@@ -24,6 +24,12 @@ void TCriExecutorConfig::Register(TRegistrar registrar)
 
     registrar.Parameter("cpu_period", &TThis::CpuPeriod)
         .Default(TDuration::MilliSeconds(100));
+
+    registrar.Parameter("retry_error_prefixes", &TThis::RetryErrorPrefixes)
+        .Default({
+            "server is not initialized yet",
+            "failed to create containerd task",
```

Comment:
Let put in the comment a link that points to concrete message from CRI codebase that we suppose to retry

For example: https://github.com/containerd/containerd/blob/53160fb4b69ea0a82fa9a050138f8b94a0927942/internal/cri/server/container_start.go#L123

Reply:
Ok, but links to issue or pr is more useful.

Actually there should be matching against more detailed error including nested errors.